js/src/modals/Verification/email-store.js

```diff
@@ -54,7 +54,7 @@ export default class EmailVerificationStore extends VerificationStore {
   }
 
   constructor (api, account, isTestnet) {
-    super(api, EmailVerificationABI, 'emailverification3', account, isTestnet);
+    super(api, EmailVerificationABI, 4, account, isTestnet);
```

Comment:
Not sure I 100% like the magic numbers (would probably make more sense as a constant to at least specify intent, especially in 3 months from now)